argilla/tests/integration/test_add_records.py

```diff
@@ -527,6 +526,60 @@ def test_add_record_resources(client):
     assert dataset_records[2].suggestions["topics"].score == [0.9, 0.8]
 
 
+def test_add_record_with_chat_field(client):
+    user_id = client.users[0].id
+    mock_dataset_name = f"test_add_record_with_chat_field{datetime.now().strftime('%Y%m%d%H%M%S')}"
+    mock_resources = [
+        rg.Record(
+            fields={
+                "chat": [
+                    {
+                        "role": "user",
+                        "content": "Hello World, how are you?",
+                    },
+                    {
+                        "role": "bot",
+                        "content": "I'm doing great, thank you!",
+                    },
+                ]
+            },
+        ),
+        rg.Record(
+            fields={
+                "chat": [
+                    {
+                        "role": "user",
+                        "content": "Hello World, how are you?",
+                    },
+                    {
+                        "role": "bot",
+                        "content": "I'm doing great, thank you!",
+                    },
+                ]
+            },
+        ),
+    ]
+    settings = rg.Settings(
+        fields=[
+            rg.ChatField(name="chat", required=True),
+        ],
+        questions=[
+            rg.TextQuestion(name="comment", use_markdown=False),
+        ],
+    )
+    dataset = rg.Dataset(
+        name=mock_dataset_name,
+        settings=settings,
+        client=client,
+    )
+    dataset.create()
+    dataset.records.log(records=mock_resources)
+
+    dataset_records = list(dataset.records(with_suggestions=True))
```

Comment:
why `with_suggestions`?
```suggestion
    dataset_records = list(dataset.records)
```